README.md

```diff
@@ -12,24 +12,21 @@ This template allows users to connect their data to DataRobot's functionality se
    - Click the blue "Add" button
 
 3. Open up a terminal (in the bottom-left corner you'll see a ">_" icon) and type the following commands.
-   - The project name does not matter.
    ```bash
-   pip install uv
-   ```
-   ```bash
-   uv pip install kedro
+   pip install kedro
    ```
    ```bash
    kedro new -n $PROJECT_NAME$ --starter=https://github.com/j-beastman/WeatherForecastic.git --checkout master
    ```
    ```bash
-   uv pip install -r $PROJECT_NAME$/requirements.txt
+   pip install -r $PROJECT_NAME$/requirements.txt
    ```
 
-4. Go into this folder, then conf/local/credentials.yml
+4. Go into this folder, then `conf/local/credentials.yml`
    - Fill in your datarobot and OpenAI credentials
+   - Note: Don't input the credentials from your notebook.
```

Comment:
Wordsmith: 
Note: Notebook environment api keys are automatically recycled. It is recommended to use a DataRobot API Token from developer tools rather than from the notebook environment.